utils/utils.go

```diff
@@ -48,3 +52,113 @@ func RemoveDuplicatesFromStringArray(arr []string) []string {
 
 	return result
 }
+
+// OutputJSONHeader holds the received Header to test sending emails (inbucket)
+type JSONMessageHeaderInbucket []struct {
+	Mailbox             string
+	ID                  string `json:"Id"`
+	From, Subject, Date string
+	To                  []string
+	Size                int
+}
+
+// OutputJSONMessage holds the received Message fto test sending emails (inbucket)
+type JSONMessageInbucket struct {
+	Mailbox             string
+	ID                  string `json:"Id"`
+	From, Subject, Date string
+	Size                int
+	Header              map[string][]string
+	Body                struct {
+		Text string
+		HTML string `json:"Html"`
+	}
+}
+
+func GetMailBox(email string) (results JSONMessageHeaderInbucket, err error) {
+
+	pos := strings.Index(email, "@")
+	parsedEmail := email[0:pos]
+
+	url := fmt.Sprintf("http://dockerhost:9000/api/v1/mailbox/%s", parsedEmail)
+
+	// Build the request
```

Comment:
Minor but can we remove this comment and other similar ones? I think the code is pretty self-explanatory